site/content/integrate/admin-guide/admin-interactive-messages/_index.md

```diff
@@ -71,6 +71,47 @@ The integration can respond with an update to the original post, or with an ephe
 
 ![image](interactive_message.gif)
 
+Button actions support a style parameter to change the color of the button. The possible values for style are: `good`, `warning`, `danger`, `default`, `primary` and `success`. It's also possible to pass a theme variable or a hex color, but we discourage this approach because it won't be resilient against theme changes.
```

Comment:
```suggestion
Button actions support a style parameter to change the color of the button. The possible values for style are: `good`, `warning`, `danger`, `default`, `primary`, and `success`. It's also possible to pass a theme variable or a hex color, but we discourage this approach because it won't be resilient against theme changes.
```